src/main/java/frc/utils/Conversions.java

```diff
@@ -0,0 +1,232 @@
+package frc.utils;
+
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import frc.robot.constants.MathConstants;
+
+public class Conversions {
+
+    public static final double MAG_TICKS = 4096;
+    public static final double DEGREES_PER_REVOLUTIONS = 360;
+    public static final double HUNDRED_MILLIESECONDS_PER_SECONDS = 10;
+    public static final double SECONDS_PER_MINUTE = 60;
+
+    /**
+     * Converts ticks from a Mag Encoder to degrees.
+     *
+     * @param magTicks ticks from a Mag Encoder
+     * @return degrees
+     */
+    public static double magTicksToDegrees(double magTicks) {
```

Comment:
add an optional conversion factor? I think about syncopa's mag encoder that wasn't 1:1..

Reply:
to all relevant functions here

Reply:
the mag encoder doesn't have 4096 ticks for sure? If i get the mag ticks also as param this function is just dividing

Reply:
I think it has, but when the mechanism doesn't have a 1:1 gear ration (like syncopa) the "number of ticks to degrees" is different, no?

Reply:
oh i understand ok